src/main/java/org/hashids/Hashids.java

```diff
@@ -197,8 +213,7 @@ public String encodeHex(String hexa) {
   /**
    * Decode string to numbers
    *
-   * @param hash
-   *          the encoded string
+   * @param hash the encoded string
```

Comment:
Respect the new line after paramater name.

Reply:
Why? This is not a JAVA rule. Perhaps a JS one.